src/assets/i18n/en-us.json

```diff
@@ -76,7 +77,7 @@
   "delivered": "Delivered",
   "returned": "Returned",
   "inProgress": "In Progress",
-  "waitingToBeAccepted": "Waiting to be Accepted",
+  "waitingToBeAccepted": "In Progress",
```

Comment:
Remove the property `"waitingToBeAccepted"`. The properties in root is for title which is capitalized to its key.